caffe2/core/operator_c10wrapper.h

```diff
@@ -28,6 +28,9 @@ using extract_type_t = typename ParameterDef::type;
  * TODO: Figure out a better way to handle output parameters
  */
 
+// TODO This wrapper shouldn't have to handle state since it's only calling into c10
+class NoState final : public c10::KernelState {};
```

Comment:
Wouldn't it make more sense to represent the NoState case as a nullptr?

Reply:
NoState is gone now